src/databricks/labs/ucx/assessment/workflows.py

```diff
@@ -183,6 +182,117 @@ def crawl_groups(self, ctx: RuntimeContext):
         ctx.group_manager.snapshot()
 
 
+class MigrationProgress(Workflow):
```

Comment:
move this to `progress` package, on the same level with assessment and source code